.github/workflows/arduino_packing_release.yml

```diff
@@ -0,0 +1,84 @@
+name: avrdude_packing
+
+env:
+  # The name of the project
+  DIST_DIR: dist
+  ARTIFACT_NAME: dist
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+*"
```

Comment:
With this you should be able to tag a release on your branch (and if it's the default one)  you should be able to make the workflow run

Reply:
Hmm, it failed.
https://github.com/mcuee/avrdude/actions/runs/8586798369

[Invalid workflow file: .github/workflows/arduino_packing_release.yml#L51](https://github.com/mcuee/avrdude/actions/runs/8586798369/workflow)
The workflow is not valid. .github/workflows/arduino_packing_release.yml (Line: 51, Col: 12): Unexpected value ''